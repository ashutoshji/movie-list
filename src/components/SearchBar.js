import React from 'react';


const SearchBar = (props) => (
    <div className="search__container">
    <p className="search__title">
        Go ahead, Search any movie
    </p>
      <input 
        className="search__input" 
        type="text" 
        value={props.searchInput} 
        onChange={props.handleChange}
        placeholder="Search for a movie"
      />
</div>
);

export default SearchBar;
