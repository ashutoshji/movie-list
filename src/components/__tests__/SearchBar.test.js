
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import SearchBar from '../SearchBar';


describe('SearchBar test', () => {
  it('SearchBar should match snapshot', () => {
    const component = renderer.create(<SearchBar
       />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
