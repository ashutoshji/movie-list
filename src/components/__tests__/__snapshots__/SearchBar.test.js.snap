// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar test SearchBar should match snapshot 1`] = `
<div
  className="search__container"
>
  <p
    className="search__title"
  >
    Go ahead, Search any movie
  </p>
  <input
    className="search__input"
    placeholder="Search for a movie"
    type="text"
  />
</div>
`;
